help/destinations/catalog/crm/salesforce.md

```diff
@@ -171,51 +186,55 @@ To correctly send your audience data from Adobe Experience Platform to the [!DNL
     * You can identify mandatory fields by searching for the word *Required*, which is mentioned in field descriptions in the link above.
     * Depending on the fields you want to export or update, add mappings between your XDM profile schema and your [!DNL Salesforce CRM] instance:
 
-    | XDM Profile Schema | [!DNL Salesforce CRM] Instance | Notes |
-    | --- | --- | --- |
-    |`person.name.lastName`|`LastName`| `Required`. Last name of the contact up to 80 characters. |    
-    |`b2b.companyName`|`Company`| `Required`. The lead's company. |
-    |`personalEmail.address`|`Email`| The contact's email address. |
+        | XDM Profile Schema | [!DNL Salesforce CRM] Instance | Notes |
+        | --- | --- | --- |
+        |`person.name.lastName`|`LastName`| `Required`. Last name of the contact up to 80 characters. |    
+        |`b2b.companyName`|`Company`| `Required`. The lead's company. |
+        |`personalEmail.address`|`Email`| The contact's email address. |
 
     * An example using these mappings is shown below:
     ![Platform UI screenshot example showing Target mappings.](../../assets/catalog/crm/salesforce/mappings-leads.png)    
 
 ### Schedule segment export and example {#schedule-segment-export-example}
 
-When performing the [Schedule segment export](/help/destinations/ui/activate-segment-streaming-destinations.md#scheduling) step you must manually map Platform segments to the custom field attribute in Salesforce.
+When performing the [Schedule segment export](/help/destinations/ui/activate-segment-streaming-destinations.md#scheduling) step you must manually map activated segments within Platform to its corresponding custom field in [!DNL Salesforce].
 
-To do this, select each segment, then enter the corresponding custom field attribute from Salesforce in the **[!UICONTROL Mapping ID]** field.
+To do this, select each segment, then enter name of the custom field from [!DNL Salesforce] in the **[!UICONTROL Mapping ID]** field. Repeat this for each activated segment within Platform.
 
 >[!IMPORTANT]
 >
->* The value used for the **[!UICONTROL Mapping ID]** should exactly match the name of the custom field attribute created within Salesforce.
->* Ensure that the name of the custom field attribute you have created in Salesforce doesn't use the whitespace character.
+>* The value used for the **[!UICONTROL Mapping ID]** should exactly match the name of the custom field created within [!DNL Salesforce].
+>* Ensure that the name of the custom field you have created in Salesforce doesn't use the whitespace character.
 
-An example is shown below:
+An example indicating the location of the Mapping ID is shown below:
 ![Platform UI screenshot example showing Schedule segment export.](../../assets/catalog/crm/salesforce/schedule-segment-export.png)
```

Comment:
Idea: Could we immediately follow up this screenshot with a screenshot of the custom field in Salesforce, showing the exact same value for the custom field in Salesforce as for the mapping ID in Platform?